config/routes.rb

```diff
@@ -14,6 +14,7 @@
   end
 
   resources :classrooms do
+    resources :grade_books, only: :show
```

Comment:
should we also add this as a resource under admin at some point so we can add/manage them?

Reply:
I haven't given this a lot of thought (I really don't have the context to say with any certainty), I think it would be good to discuss some of the business requirements here and come up with a plan for sure though.